Configuration/TypoScript/setup.typoscript

```diff
@@ -25,37 +25,36 @@ plugin.tx_form {
 	)
 }
 
-plugin.tx_bwcaptcha {
+module.tx_bwcaptcha {
```

Comment:
Why are you switching from plugin to module? The module key is normaly used for backend relevant TypoScript.

Reply:
You are right, mostly for backend but also for non-plugin TypoScript like middlewares or CLI applications.
So you just changed "module" back to "plugin" in TS?
Actually that should then fall back to default settings and not respect your typoscript configuration.
My middleware only parses configs from "module".

Reply:
That's why I changed the identifier back to plugin in your middleware as well. The middleware is called via an asynchronous request - so we are still in the frontend. The project I tested your pull request in (TYPO3 v11.5.24), the middleware is able to read the TypoScript data from plugin.tx_bwcaptcha...

And yes, my custom TypoSript settings and not just the default values. Have you tested my pull request?